### DISCUSSION TITLE
Feature Request: Remove many keys

### DISCUSSION THREAD

**Discussion by tomhobson** [ekvedaras/redis-gui#41] [Ideas]
Hey, thanks for making this tool, super useful!

It would be great if there was a means to delete many keys from the gui. 

If I want to flush the cache or something for a specific service or a specific set of keys, I don't really have a means to do that

**Comment by ekvedaras** [comment]
Will see what I can do. For now, try this: https://rdbtools.com/blog/redis-delete-keys-matching-pattern-using-scan/
